client/my-sites/plugins/single-plugin.jsx

```diff
@@ -322,7 +302,7 @@ function CTA( { slug, isPluginInstalledOnsite, fullPlugin } ) {
 		return (
 			<Button
 				className="single-plugin__install-button"
-				onClick={ () => onClickInstallPlugin( { dispatch, selectedSiteId, fullPlugin, slug } ) }
+				href={ `/marketplace/${ slug }/install/${ selectedSite.slug }` }
```

Comment:
We will need to use `onClick` both here and in https://github.com/Automattic/wp-calypso/pull/57478/commits/f10698fd4e9b34a714857b83bc1491595dbe0a20#diff-fbb59cfe300fa8fdfab594360525688a332b9319de457754a6b79f4266bf8d2cR315 so that we still dispatch the "click" track events.

```
dispatch( recordGoogleEvent( 'Plugins', 'Install on selected Site', 'Plugin Name', slug ) );
	dispatch(
		recordGoogleEvent( 'calypso_plugin_install_click_from_plugin_info', {
			site: selectedSiteId,
			plugin: slug,
		} )
	);
```

Reply:
Cool, will make generalize the code for both options and send the events.